docs/developer.rst

```diff
@@ -65,6 +65,47 @@ To build ``jaxlib`` without CUDA GPU support (CPU only), drop the ``--enable_cud
   python build/build.py
   pip install -e build  # installs jaxlib (includes XLA)
 
+
+Additional Notes for Building ``jaxlib`` from source on Windows
+...............................................................
+
+On Windows, following `Install Visual Studio <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`_
+to setup latest C++ toolchain. If you need to build with CUDA enabled, follow
+`CUDA Installation Guide <https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html>`_
+to setup CUDA environment.
+
+Install `bazelisk <https://github.com/bazelbuild/bazelisk/releases/latest>`_.
+Download and rename it as ``bazel.exe`` and add it to your path.
+
+It is recommended to use `Anaconda <https://docs.anaconda.com/anaconda/install/windows/>`_
+or `Miniconda <https://docs.conda.io/en/latest/miniconda.html#windows-installers>`_
+to setup python environment.
+
+Some targets of Bazel use bash utilities to do scripting, so `MSYS2 <https://www.msys2.org>`_
+is needed. See `Installing Bazel on Windows <https://docs.bazel.build/versions/master/install-windows.html#installing-compilers-and-language-runtimes>`_
+for more details. Install the following packages::
+
+  pacman -S patch realpath
+
+
+Once everything is installed. Open PowerShell, make sure you've added MSYS2
+path to current session. Ensure ``bazel``, ``patch`` and ``realpath`` are
+accessible. Activate conda environment. Folllowing command build with cuda
+enabled with PDB generated, adjust it to whatever suitable for you.::
+
+  python .\build\build.py `
+    --enable_cuda `
+    --cuda_path='C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.1' `
+    --cudnn_path='C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.1' `
+    --cuda_compute_capabilities='6.1' `
+    --cuda_version='10.1' `
+    --cudnn_version='7.6.5' `
+    --bazel_options='--copt' `
```

Comment:
I think I would leave these out by default, and instead I would say something like "To build with debug information, add the flag: ``--bazel_options=--copt=/Z7``."

Reply:
These version things seems to be need currently. Blame bazel xD